### PR TITLE
Universal test for prebuild step

### PR DESCRIPTION
**Pull request by martint-unity**
---
### Purpose of this PR
To have a place to do pre build things

### CODE REVIEW

File: TestProjects/UniversalGraphicsTest/Assets/Test/Editor/PreBuildStep.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+public class PreBuildStep : IPrebuildSetup
+{
+    List<string> ScenesToNotBuild = new List<string> {"035_Shader_TerrainShaders", };
```
Comment (nigeljw-unity) [inline TestProjects/UniversalGraphicsTest/Assets/Test/Editor/PreBuildStep.cs:9]: Do we also want 132_DetailMapping?

Comment (nigeljw-unity) [reply TestProjects/UniversalGraphicsTest/Assets/Test/Editor/PreBuildStep.cs:9]: Looking at the remaining failures in the tests that have run so far, it also looks like Linux_Vulkan_playmode_cache_mono_Linear for 035_Shader_TerrainShader is also still failing?

Comment (nigeljw-unity) [reply TestProjects/UniversalGraphicsTest/Assets/Test/Editor/PreBuildStep.cs:9]: Detail mapping seems to only be for Linux playmode as well. The change seems like it worked:
https://yamato-artifactviewer.prd.cds.internal.unity3d.com/16e172d9-58c4-4531-b316-5b618e0f6495%2FTest_report%2FUtrTestReport2%2FTestProjects%2FUniversalGraphicsTest%2Ftest-results/TestReportV1.html

Comment (martint-unity) [reply TestProjects/UniversalGraphicsTest/Assets/Test/Editor/PreBuildStep.cs]: I had the 132_detailmapping in my commit, just forgot to push last night. doh! :)

**Comment by martint-unity** [comment]
Failures are known in trunk.

**Review by nigeljw-unity — APPROVED** [review]

**Pull request merged**